tool/src/js/mod.rs

```diff
@@ -32,12 +32,23 @@ impl FileType {
 pub(crate) fn attr_support() -> BackendAttrSupport {
     let mut a = BackendAttrSupport::default();
 
+    a.renaming = true;
+    a.namespacing = false;
     a.memory_sharing = false;
     a.non_exhaustive_structs = true;
     a.method_overloading = false;
     a.utf8_strings = false;
     a.utf16_strings = true;
+
+    a.constructors = false;
+    a.named_constructors = false;
     a.fallible_constructors = false;
+    a.accessors = true;
+    a.comparators = false;
+    a.stringifiers = false; // TODO
+    a.iterators = false; // TODO
+    a.iterables = false; // TODO
```

Comment:
I think one of these was wrong, JS started producing iterators once they weren't cfged anymore.

Reply:
@ambiguousname says that the iterator support in JS is partial and broken

Reply:
Yeah, it's in a branch back at home that I have yet to publish, I will open a PR when that's ready to review

Reply:
Iterators are *sort of* there, but if you try to use them they will not work

Reply:
(and this is partly why I don't want diplomat to feed backends attributes it doesn't support: one of the primary goals of attributes was to enable backends to be developed by different people and diverge in how they behave whilst using the same underlying framework, and attributes give diplomat_core a level of control over that interplay without forcing diplomat_core devs to accidentally inadvertently affect these backends too much)